prow/cmd/deck/static/tidescript.js

```diff
@@ -71,22 +71,37 @@ function createLinkCell(text, url, title) {
     return c;
 }
 
-function createTextCell(text) {
+function createActionCell(pool) {
+    var action = pool.Action;
+    var targetted = pool.Target && pool.Target.length
     var c = document.createElement("td");
-    c.appendChild(document.createTextNode(text));
+
+    if (targetted) {
+        action += ": "
+    }
+    c.appendChild(document.createTextNode(action));
+    if (targetted) {
+        addPRsToElem(c, pool, pool.Target)
+    }
     return c;
 }
 
 function createPRCell(pool, prs) {
     var c = document.createElement("td");
+    addPRsToElem(c, pool, prs)
+    return c;
+}
+
+function addPRsToElem(elem, pool, prs) {
     if (prs) {
         for (var i = 0; i < prs.length; i++) {
             var a = document.createElement("a");
             a.href = "https://github.com/" + pool.Org + "/" + pool.Repo + "/pull/" + prs[i].Number;
             a.appendChild(document.createTextNode("#" + prs[i].Number));
-            c.appendChild(a);
-            c.appendChild(document.createTextNode(" "));
+            elem.appendChild(a);
+            if (i+1 < prs.length) {
+                elem.appendChild(document.createTextNode(" "));
```

Comment:
why exactly are we appending a " " node?

Reply:
Yeah, add a comment here. The logic is tricky.

Reply:
Its a space between the PR number links.

Reply:
I suppose not. Maybe add a comment breifly mentioning it what way this adds PRs to the element? "adds a list of space separated PR links". The method name is terse for no comments.

Reply:
I added some comments.